serde_tests/tests/test.rs

```diff
@@ -1,6 +1,4 @@
 #![cfg_attr(feature = "nightly", feature(plugin))]
 #![cfg_attr(feature = "nightly", plugin(clippy))]
 
-extern crate serde;
```

Comment:
No error, just to demonstrate that things work even without "extern crate serde" at the top level. I kept the equivalent line in [bench.rs](https://github.com/dtolnay/serde/blob/5eb1d7dd4b9d5aa19599be10771fb020a39c46a3/serde_tests/benches/bench.rs#L6) so the build will still uses both approaches.